### DISCUSSION TITLE
客户端可以用代码创建库和表吗？(Can Users create databases/tables using code)

### DISCUSSION THREAD

**Discussion by dong-lufei** [supabase/supabase#6883] [Questions] [answered]
客户端可以用代码创建数据库和数据表吗？而不是在控制台界面手工创建或sql编辑器里面运行sql

**Comment by J0** [comment, marked as answer]
可以,看一下[Postgrest-js](https://github.com/supabase/postgrest-js)


Loose translation for non speaking users(forgive my Chinese):

 Q: Can users make use  of code to create Databases or tables  without using the control panel to construct a query or executing via sql client.

A: Yes, take a look at postgrest-js which exposes Postgres as a RESTful API which users can perform operations against.
